### DISCUSSION TITLE
Connection REFUSED port 25

### DISCUSSION THREAD

**Discussion by gigbuilder** [postalserver/postal#2350] [Help with using Postal]
I've installed twice -- both on clean CENTOS7 servers.  I removed firewalld - did the entire install - setup routes - yet cannot telnet to hostname 25 --> I can telnet locally into localhost via localhost but cannot telnet with the domain or telnet any other way.
I own the server -- no ports are blocked
What is going on?

**Comment by willpower232** [comment]
You can check on the server with `netstat -lnp` that Postal is definitely listening on port 25 and if it is then there must be something else blocking the ports either remotely or locally.

Some cloud server providers have their own firewall thing which could be involved. Alternatively, your own internet provider could be blocking that port too which I think has happened before.

**Reply by gigbuilder** [reply]
I just figured it out -- postfix was still installed -- systemctl stop postfix -- yum remove postfix -- I'll leave this here for reference.
